### PR TITLE
Replace isset with get

### PR DESCRIPTION
**Pull request by dshanske**
Ask and ye shall receive

### CODE REVIEW

**Comment by snarfed** [comment]
ahhh this is so good! thank you!!! mind switching all of the existing `isset`s to it? not urgent.

i'm also tempted to move it out of the class, put i guess that puts it in the global PHP namespace, which is a bad idea...? yeah probably never mind. :P

**Comment by snarfed** [comment]
friendly ping! this would be great to get in.

**Comment by dshanske** [comment]
We can put it into the global namespace if we prefix it

**Comment by snarfed** [comment]
eh, i'm fine with it as a static method, unless you feel strongly otherwise.

**Comment by dshanske** [comment]
@snarfed I cleaned out a bunch of get related errors with undefined indexes and changed the signature of the get function to be more array index friendly. 

Also found the same variable name was used in two different contexts by accident, so changed it.

I did make one change that isn't strictly in the PR notification. I changed serialized in the error_log to json_encode because I found it easier to read. I think I originally put the serialize in.

File: micropub.php [hunk 1]
```diff
@@ -925,28 +932,27 @@ public static function store_micropub_auth_response( $args ) {
 	 */
 	public static function store_mf2( $args ) {
 		$props = static::$input['properties'];
-		if ( $props ) {
-			if ( ! isset( $args['meta_input'] ) ) {
-				$args['meta_input'] = array();
-			}
-			$type = static::$input['type'];
+		if ( ! isset( $args['ID'] ) && $props ) {
```
Comment (snarfed) [inline micropub.php]: why was `! isset( $args['ID'] )` added here? seems unrelated?

**Review by snarfed — CHANGES_REQUESTED** [review]
hey, thanks! lgtm other than one errant change.

File: micropub.php (hunk 1 above)
Comment (dshanske) [reply micropub.php]: Because before, it would run the remaining lines, which don't apply on creation, generating an error.

Comment (dshanske) [reply micropub.php]: When I went to turn it into get, I realized that it was trying to run the replacement code on create.

Comment (snarfed) [reply micropub.php]: you mean it was running the create code on replace? (and delete and add?) i think that was ok as is, but this is fine too. ok!

**Review by snarfed — APPROVED** [review]
